package.json

```diff
@@ -16,6 +16,7 @@
     "@mui/x-data-grid": "^5.6.1",
     "@walletconnect/web3-provider": "^1.7.7",
     "@web3auth/web3auth": "^0.7.0",
+    "ethers": "^5.6.4",
```

Comment:
`package-lock.json` again has a 10k line diff, `npm i` says that it's because you're using an old version of npm. I'll fix on my end and push.